app/Platform/Controllers/DeveloperGameListController.php

```diff
@@ -0,0 +1,191 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Platform\Controllers;
+
+use App\Community\Enums\TicketState;
+use App\Community\Models\Ticket;
+use App\Http\Controller;
+use App\Platform\Models\Game;
+use App\Platform\Models\PlayerGame;
+use App\Platform\Models\System;
+use App\Site\Models\User;
+use Illuminate\Contracts\View\View;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
+
+class DeveloperGameListController extends Controller
+{
+    public function __invoke(Request $request): View
+    {
+        $username = $request->route('user');
+        $user = User::firstWhere('User', $username);
+        if ($user === null) {
+            abort(404);
+        }
```

Comment:
When attempting to access a banned user's page, a division by zero error is thrown. May be worth adding something like a `getCanViewTargetUser()` function similar to the completion progress page.

Reply:
I suspect you're using a very specific banned user, who has a set worth 0 points. Good find. I've corrected for that.

Banned users still have accessible profiles, so their list of dev'd sets should still be accessible as well.

Reply:
Yes :)

Got it - I actually did not realize those dev profiles were still accessible (I assumed they weren't). Marking as resolved.